.github/workflows/ci.yaml

```diff
@@ -136,4 +136,4 @@ jobs:
           --enforce-https \
           --external-only \
           --url-swap "^^https\://alan-turing-institute.github.io/REG-handbook/:/,/edit/main:/edit/$(git branch --show-current),/tree/main:/tree/$(git branch --show-current)" \
-          --url-ignore "/github.com/alan-turing-institute/research-engineering-group/,/docs.github.com/"
```

Comment:
Was this for one repo in particular? If it is a small number of repos it might be good to list them explicitly.